nginx-php-mysql/src/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
I would add this file to the backend directory.

Reply:
I had initially had it this way but wasn't sure if it was too confusing for some given that both nginx and php files will go here. I thought SRC was a better name given that it would contain both frontend and backend files.

Reply:
@aiordache this has been updated in the latest commit.